pkgs/top-level/all-packages.nix

```diff
@@ -35239,18 +35239,18 @@ with pkgs;
 
   syncthing-tray = callPackage ../applications/misc/syncthing-tray { };
 
-  syncthingtray = libsForQt5.callPackage ../applications/misc/syncthingtray { };
-  syncthingtray-minimal = libsForQt5.callPackage ../applications/misc/syncthingtray {
+  syncthingtray = kdePackages.callPackage ../applications/misc/syncthingtray {
+    # renamed in KF5 -> KF6
+    plasma-framework = kdePackages.libplasma;
+  };
+  syncthingtray-minimal = syncthingtray.override {
     webviewSupport = false;
     jsSupport = false;
     kioPluginSupport = false;
     plasmoidSupport = false;
     systemdSupport = true;
   };
-  syncthingtray-qt6 = kdePackages.callPackage ../applications/misc/syncthingtray {
-    # renamed in KF5 -> KF6
-    plasma-framework = kdePackages.libplasma;
-  };
+  syncthingtray-qt5 = libsForQt5.callPackage ../applications/misc/syncthingtray { };
```

Comment:
Is this really needed?

Reply:
TBH I haven't thought about a lot, but I think there are many people out there who don't use KDE and don't have a lot of QT packages, and the packages they use don't support qt6 yet, and they'd wish to not depend on qt6 if they don't have to. OTH, adding a new qt5 attribute could be going against progress, so I won't mind removing it.

Reply:
Let's remove it. There are many other packages that only use Qt 5 despite supporting both.

Reply:
> Let's remove it. There are many other packages that only use Qt 5 despite supporting both.

:+1:.